recipe_scrapers/forksoverknives.py

```diff
@@ -1,13 +1,44 @@
 # mypy: disallow_untyped_defs=False
 from ._abstract import AbstractScraper
 from ._utils import normalize_string
+from ._schemaorg import SchemaOrg
+from typing import Dict, List, Optional, Tuple, Union
+
+import re, html
+
+class fok_Schema(SchemaOrg):
+    def __init__(self, page_data, raw=False):
+        super().__init__(page_data)
+
+    def category(self):
+        cat = super().category()
```

Comment:
Thanks for the fix @manithree!

It would probably be simpler if we can avoid adding the `fok_Schema` class - maybe something like the code shown below?

```python
class ForksOverKnives(AbstractScraper):
...
    def category(self):
        cat = self.schema.category()
        r = re.compile('.*<a href="\S+">(.*)</a>')
        ....
```

(does that work?  I think it should, but I haven't tried it out locally yet.  that way, we could also avoid adding the `__init__` constructor override on the class)

Reply:
I'm pretty sure I tried that (I didn't keep very good notes in the learning process), but my goal is to get this working in gourmand again, and [this line of code in gourmand](https://github.com/GourmandRecipeManager/gourmand/blob/928b6962e8dd49fe011b55eddf34d92324c29885/src/gourmand/importers/web_importer.py#L115) is why I did it the way I did.
I didn't write that line of code, and I don't know if it's a "correct" or idiomatic usage of the API. But if this PR is rejected for doing the inheritance, that at least gives me a starting point for a conversation with the gourmand maintainer(s).

Reply:
Ah, good to know - thanks for the context - yep, it looks like `prep_time`, `cook_time` and `category` all all being retrieved via the `scraper.schema` object.

Tricky - I think calling the methods on the scraper directly would be easier for us, maintenance-wise (less complex).  If you don't mind finding out a bit more from the Gourmand folks, that'd be great (cc @cydanil as you're a contributor on both projects).

(hopefully we don't have to completely reject this change, but if there's a better way, it'd be good to try to discover that)

Reply:
I've filed a (probably not very well described) issue at GourmandRecipeManager/gourmand#132 for the method-call oddity/workaround, so that we can track that.  Thanks for explaining the issue.

Reply:
That'd make sense, thanks.  Hmm.. I'm not completely sure how best to resolve that.

Maybe simply adding the name `category` to the [`schema.org` plugin's method list](https://github.com/hhursev/recipe-scrapers/blob/05a04a8e2b17297a774bb9f851c4a0e93e1d20ff/recipe_scrapers/plugins/schemaorg_fill.py#L21-L34) on our side would fix this?  (not tested yet, but I think that might resolve the problem)

Reply:
Possible fix opened as #705

Reply:
#705 was merged into Gourmand, so hopefully from their next released version onwards, we may not need this workaround for the `category` method calls.

The next question is whether to wait for an upcoming release of Gourmand, or whether to merge these changes before then (and then apply cleanups after a release appears).

Reply:
I think you shouldn't wait for the next release of Gourmand as  
1) I'm aware of the situation  
2) There's a fix in place, and we're not not dynamically updating this package (yet).  
3) The next release of Gourmand will probably not be sooner than late February.  

I appreciate the consideration though!

Reply:
Ok, thanks @cydanil!